### PR TITLE
Add Unity

### PR DESCRIPTION
**Pull request by leereilly**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [ ] Suggesting edits to an existing Topic page
  - [x] Curating a new Topic page

************CURATING A NEW TOPIC PAGE************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

There are about [3k projects tagged wth `unity`](https://github.com/topics/unity) already, plus a gazillion more that aren't tagged.

### CODE REVIEW

File: topics/unity/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+aliases: unity3d
+created_by: Unity Technologies
+display_name: Unity
+github_url: https://github.com/Unity-Technologies
+logo: unity.png
+short_description: Unity is a game development platform used to build high-quality 3D, and 2D games, deploy them across mobile, desktop, VR/AR, consoles, or the Web.
+topic: unity
+url: https://unity3d.com/
+---
+Unity is a game development platform used to build high-quality 3D, and 2D games, deploy them across mobile, desktop, VR/AR, consoles, or the Web.
```
Comment (cheshire137) [inline topics/unity/index.md]: I think the comma before "and 2D" is unnecessary.

It feels a bit odd to capitalize "the Web".

What do you think about spelling out VR and AR?

File: topics/unity/index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+aliases: unity3d
+created_by: Unity Technologies
+display_name: Unity
+github_url: https://github.com/Unity-Technologies
+logo: unity.png
+short_description: Unity is a game development platform used to build high-quality 3D, and 2D games, deploy them across mobile, desktop, VR/AR, consoles, or the Web.
```
Comment (cheshire137) [inline topics/unity/index.md]: The comma before "and 2D" seems unnecessary.

Can you rephrase the short_description to be different from the description? Maybe rework the full description below to have more details than this short_description.

Comment (leereilly) [reply topics/unity/index.md]: I agree - I made that change to make the Oxford command test go from 🔴  to 💚 :

```
  1) Failure:
topics::unity topic#test_0014_follows the Topic Page Style Guide [/Users/leereilly/github/explore/test/topics_test.rb:207]:
Always use the Oxford comma.
Expected "Unity is a game development platform used to build high-quality 3D and 2D games, deploy them across mobile, desktop, VR/AR, consoles, or the Web" to include ", and".
```

File: topics/unity/index.md (hunk 1 above)
Comment (leereilly) [reply topics/unity/index.md]: Made the suggested changes - thanks! ✓ 

> What do you think about spelling out VR and AR?

I think VR/AR is generally well understood... and I prefer the short version here. That said, it we have any hard and fast rules or general preferences I can add it in parenthesis or use some `<acronym>` tags. Let me know 🤘
